packages/e2e-tests/specs/editor/plugins/meta-boxes.test.js

```diff
@@ -60,7 +60,7 @@ describe( 'Meta boxes', () => {
 		await page.waitForNavigation();
 
 		// Check the the dynamic block appears.
-		await page.waitForSelector( '.wp-block-latest-posts' );
+		await page.waitForSelector( '.wp-block-latest-posts__list' );
```

Comment:
I feel like the absence of the `.wp-block-latest-posts` class is an actual bug that the failing test has caught rather than a problem with the test itself, but I could be wrong. I don't really know the background of the change that caused it.

Reply:
I just updated the description of the PR with my reasoning (sorry I jumped the gun a bit!). I couldn't reproduce the problem outside of the e2e testing environment, so I have a hunch it might be a false positive.

Reply:
Thanks for following up on it, definitely better to be proactive 😄 

I've made an issue for the bug https://github.com/WordPress/gutenberg/issues/26441

Reply:
Thanks for the quick replies Daniel. I'll go ahead and close this PR for now 🙂